test/CMakeLists.txt

```diff
@@ -14,3 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+include(FindGTestAlt)
```

Comment:
Do we actually need a separate FindGTestAlt file?

Reply:
I have no strong opinion here, I just thought gathering all the thirdparties' cmake file in cmake_modules might be easy to manage. I admit this is a simple one which could be insert directly here, so will change to that.